ProcessMaker/screen-builder
# Add option to get collection as a data source for select lists

Add new dropdown option and related data fetching methods

Required branches/PRs
- https://github.com/ProcessMaker/package-collections/pull/335
- https://github.com/ProcessMaker/package-webentry/pull/186
- https://github.com/ProcessMaker/vue-form-elements/pull/381
- https://github.com/ProcessMaker/processmaker/pull/4659

## How to Test
- Add a collection with example data and ensure you can use it as a source for the select list options

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-7437

## Code Review Checklist

.
